### PR TITLE
Added new fields to the profiles table

### PR DESCRIPTION
**Pull request by Mawadddah**

### CODE REVIEW

File: server/db/recreate-schema.sql [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
-DROP TABLE if exists users;
 DROP TABLE if exists profiles;
+DROP TABLE if exists users;
 DROP TYPE if exists gender;
+DROP TYPE if exists type;
+DROP TYPE if exists status;
 
 -- Custom Types
 CREATE TYPE gender AS ENUM ('female', 'male', 'other', 'not_provided');
+CREATE TYPE type AS ENUM ('volunteer', 'service_user');
```
Comment (tekul) [inline server/db/recreate-schema.sql]: Maybe call this `profile_type` rather than just `type`.

File: server/db/recreate-schema.sql [hunk 2]
```diff
@@ -13,6 +18,8 @@ CREATE TABLE users (
   password VARCHAR(200) NOT NULL
 );
 
+
+
```
Comment (tekul) [inline server/db/recreate-schema.sql]: Could you delete this extra space? Try not to add random space in files :slightly_smiling_face:.

File: server/db/recreate-schema.sql [hunk 3]
```diff
@@ -22,5 +29,10 @@ CREATE TABLE profiles (
   email           VARCHAR(200) NOT NULL UNIQUE,
   address         VARCHAR(200),
   phone_number    VARCHAR(50),
-  occupation      VARCHAR(100)
+  occupation      VARCHAR(100),
+  type            type NOT NULL,
+  status          status,
+  join_date       DATE,
```
Comment (tekul) [inline server/db/recreate-schema.sql]: The `status` and `join_date` columns should probably be `NOT NULL` too.

**Review by tekul — CHANGES_REQUESTED** [review]
Nice work :smile_cat: :1st_place_medal: .

**Review by bonboh — APPROVED** [review]
Great job @AaishaMohamad!

**Pull request merged**